frontend/src/components/general/TaskStyles.js

```diff
@@ -0,0 +1,33 @@
+import {
```

Comment:
TaskStyles to trochę niefortunna nazwa dla tego pliku bo bardziej sugeruje plik styled-components